src/platforms/dotnet/guides/uwp/index.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: UWP
+description: "Learn more about Sentry's .NET SDK works with Universal Windows platform (UWP) applications."
+redirect_from:
+  - /platforms/dotnet/uwp/
+---
+
+Sentry's .NET SDK works with Universal Windows platform applications through the [Sentry NuGet package](https://www.nuget.org/packages/Sentry). It works with UWP apps running on .NET Native.
+
+## Configuration
+
+In addition to [initializing the SDK with `SentrySdk.Init`](/platforms/dotnet/), you must configure the UWP specific error handler.
+
+<Note>
+
+The SDK should be initialized in the constructor of your application class (usually `App.xaml.cs`). Do not initialize the SDK in the `OnLaunched()` event of the application or the `Hub` will not be initialized correctly.
+
+</Note>
+
+```csharp
+using Sentry;
+using System.Runtime.ExceptionServices;
+using System.Security;
+using Windows.UI.Xaml;
+using UnhandledExceptionEventArgs = Windows.UI.Xaml.UnhandledExceptionEventArgs;
+
+
+sealed partial class App : Application
+{
+    public App()
+    {
+        SentrySdk.Init("___PUBLIC_DSN___");
+        Current.UnhandledException += ExceptionHandler;
+    }
+
+    [HandleProcessCorruptedStateExceptions, SecurityCritical]
+    internal void ExceptionHandler(object sender, UnhandledExceptionEventArgs e)
+    {
+        //We need to backup the reference, because the Exception reference last for one access.
+        //After that, a new  Exception reference is going to be set into e.Exception.
+        var exception = e.Exception;
+        if (exception != null)
+        {
+            exception.Data[Mechanism.HandledKey] = false;
+            exception.Data[Mechanism.MechanismKey] = "Application.UnhandledException";
+            SentrySdk.CaptureException(exception);
+            //If you are not going to use the Cache functionality, it's recommended to flush 
+            //Sentry for forcing it to send the Exception before the app closes.
+            //  SentrySdk.FlushAsync(TimeSpan.FromSeconds(10)).Wait();
```

Comment:
```suggestion
        // We need to hold the reference, because the Exception property is cleared when accessed
        var exception = e.Exception;
        if (exception != null)
        {
            // Tells Sentry this was an Unhandled Exception
            exception.Data[Mechanism.HandledKey] = false;
            exception.Data[Mechanism.MechanismKey] = "Application.UnhandledException";
            SentrySdk.CaptureException(exception);
            // Make sure the event is flushed to disk or to Sentry
            SentrySdk.FlushAsync(TimeSpan.FromSeconds(3)).Wait();
```